src/map/skill.c

```diff
@@ -11096,6 +11243,75 @@ static int skill_castend_nodamage_id(struct block_list *src, struct block_list *
 			break;
 		case SO_ELEMENTAL_SHIELD:/* somehow its handled outside this switch, so we need a empty case otherwise default would be triggered. */
 			break;
+		case RL_RICHS_COIN:
+			if (sd) {
+				int i;
+
+				clif->skill_nodamage(src, bl, skill_id, skill_lv, 1);
+				for (i = 0; i < 10; i++)
```

Comment:
should be ``for (int i = 0; i < ...``

Reply:
lol yeah. i was not into such but since only loop uses it. that should be the case.